app/lib/mongo_exporter.rb

```diff
@@ -21,7 +21,7 @@ def self.export_all(path:)
     end
   end
 
-  def self.execute(cmd, *args)
-    system cmd, *args
+  def self.execute(*args)
+    `#{args.join(" ")}`
```

Comment:
Ah brakeman doesn't like this, which I guess makes total sense. I'd probably just drop this function and do the .join inline — that way at least if you have to suppress the warning it'll be easier to see that the code is ok and there's no chance of some poor future soul calling this function with untrusted inputs.

Reply:
Yeah, I found a Brakeman-friendly way of piping commands together with `Open3.pipeline(cmd1, cmd2, ...)` and changed it to that in [3ae3c99](https://github.com/alphagov/content-store/pull/1145/commits/3ae3c99abf4c952f98f950d10b7cb392e4378811)